res/css/_common.pcss

```diff
@@ -484,13 +483,24 @@ legend {
 
 .mx_Dialog_cancelButton {
     @mixin customisedCancelButton;
+    position: relative;
     width: 18px;
     height: 18px;
     position: absolute;
     top: 10px;
     right: 0;
 }
 
+.mx_Dialog_cancelButton::before{
+    content: "";
+    position: absolute;
+    height: 50px;
```

Comment:
Ideally we should avoid hardcoding the height but use `inset` or a combination of `top` and `bottom` so that the height is inferred from the parent element.

More future proof.

Reply:
As per your suggestion, Using top and bottom added the size for the tocuh area. so you can check below area of touch area for this cancel button

<img width="74" alt="image" src="https://github.com/matrix-org/matrix-react-sdk/assets/36238382/9a10ba9e-de75-4cde-ad83-99755a49f861">